modules/ossm-about-multi-cluster-topologies.adoc

```diff
@@ -0,0 +1,33 @@
+// This module is used in the following assemblies:
+
+// * install/ossm-multi-cluster-topology.adoc
+
+:_mod-docs-content-type: Concept
```

Comment:
Technically these are supposed to be allcaps:
```suggestion
:_mod-docs-content-type: CONCEPT
```

Reply:
done